src/lib/structures/Store.ts

```diff
@@ -65,10 +66,11 @@ export class Store<T extends Piece> extends Collection<string, T> {
 	}
 
 	/**
-	 * The extras to be used for dependency injection in all pieces. Returns a reference of [[Store.defaultExtras]].
+	 * A reference to the [[container]] object for ease of use.
```

Comment:
```suggestion
	 * A reference to the {@link container} object for ease of use.
```

![image](https://user-images.githubusercontent.com/4019718/117540265-accbcc00-b00e-11eb-85da-0105c56076a9.png)